src/basics.md

```diff
@@ -137,7 +137,7 @@ fn main() {
     // Assume the crawler returned the following results
     let crawler_results = &[
         DiscoveredItem {
-            blog_url: "https://blogs.com/".to_owned().to_owned(),
+            blog_url: "https://blogs.com/".to_owned(),
```

Comment:
@a1akris That the typo fixed. The rest of the changes is the result of running `mdbook build -d docs`.

Reply:
Thx!